cmd/create/machinepool/machinepool.go

```diff
@@ -253,6 +277,47 @@ func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster
 		}
 	}
 
+	securityGroupIds := args.securityGroupIds
+	if interactive.Enabled() && isVersionCompatibleComputeSGIds && isByoVpc {
+		var err error
+		availableSubnets, err := r.AWSClient.GetVPCSubnets(cluster.AWS().SubnetIDs()[0])
+		if err != nil {
+			r.Reporter.Errorf("Failed to retrieve available subnets: %v", err)
+			os.Exit(1)
+		}
+		firstSubnet := availableSubnets[0]
+		vpcId := *firstSubnet.VpcId
```

Comment:
is the VpcId or subnet a pointer? If it's the VpcId i think it would be safe to include a nil check in the if statement below

Reply:
i couldnt find `GetVPCSubnets` in CS so i am unable to check myself